source/about/mattermost-v10-changelog.md

```diff
@@ -7,6 +7,78 @@ Support for Mattermost Server v9.5 [Extended Support Release](https://docs.matte
 - See the [changelog in progress](https://bit.ly/2nK3cVf) for details about the upcoming release.
 ```
 
+(release-v10.3-feature-release)=
+## Release v10.3 - [Feature Release](https://docs.mattermost.com/about/release-policy.html#release-types)
+
+**Release Day: December 16, 2024**
+
+### Important Upgrade Notes
+
+ - As part of our continuous efforts to enhance your user experience, we have phased out the Classic Mobile App. Please download the new v2 Mobile App from the [Apple App Store](https://apps.apple.com/us/app/mattermost/id1257222717) or [Google Play Store](https://play.google.com/store/apps/details?id=com.mattermost.rn) to continue enjoying our services without interruption. See more details in the [classic mobile app deprecation](https://forum.mattermost.com/t/classic-mobile-app-deprecation/18703) Mattermost forum post.
```

Comment:
nit: The introductory phrasing has a "corporate vibe" to it. Would we consider just omitting it, stating the changes, and linking to the forum post?